docs/includes/register-sso-add-in-aad-v2-include.md

```diff
@@ -1,26 +1,11 @@
+### Create the app registration
 
-1. Navigate to the [Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page to register your app.
+First, complete the steps in [Quickstart: Register an application with the Microsoft identity platform](/azure/active-directory/develop/quickstart-register-app) to create an initial app registration. After you complete the step [Add credentials](/azure/active-directory/develop/quickstart-register-app#add-credentials) return to this article and continue following the steps in [Expose a web API](#expose-a-web-api).
```

Comment:
Whoa. If you make these changes, the include is only relevant to projects created with Yo Office. What about those created with Visual Studio?